docs/list_of_checks.rst

```diff
@@ -0,0 +1,431 @@
+       -> Description of the container displayed in Kubernetes
+       -> https://github.com/projectatomic/ContainerApplicationGenericLabels/blob/master/vendor/redhat/labels.md
+       -> ['io.k8s.description', 'description', 'label', 'required']
+       -> required
+
+    architecture_label_capital_deprecated
+       -> Label 'Architecture' is deprecated.
+       -> Replace with 'architecture'.
+       -> ?????
+       -> ['architecture', 'label', 'capital', 'deprecated']
+       -> required
+
+    bzcomponent_deprecated
+       -> Label 'BZComponent' is deprecated.
+       -> Replace with 'com.redhat.component'.
+       -> ?????
+       -> ['com.redhat.component', 'bzcomponent', 'label', 'deprecated']
+       -> required
+
+    name_label_capital_deprecated
+       -> Label 'Name' is deprecated.
+       -> Replace with 'name'.
+       -> ?????
+       -> ['name', 'label', 'capital', 'deprecated']
+       -> required
+
+    version_label_capital_deprecated
+       -> Label 'Version' is deprecated.
+       -> Replace with 'version'.
+       -> ?????
+       -> ['version', 'label', 'capital', 'deprecated']
+       -> required
+
+    install_label_capital_deprecated
+       -> Label 'INSTALL' is deprecated.
+       -> Replace with 'install'.
+       -> ?????
+       -> ['install', 'label', 'capital', 'deprecated']
+       -> required
+
+    uninstall_label_capital_deprecated
+       -> Label 'UNINSTALL' is deprecated.
+       -> Replace with 'uninstall'.
+       -> ?????
+       -> ['uninstall', 'label', 'capital', 'deprecated']
+       -> required
+
+    release_label_capital_deprecated
+       -> Label 'Release' is deprecated.
+       -> Replace with 'release'.
+       -> ?????
+       -> ['release', 'label', 'capital', 'deprecated']
+       -> required
+
+    vcs-url_label
+       -> Label 'vcs-url' has to be specified.
+       -> URL of the version control repository.
+       -> https://github.com/projectatomic/ContainerApplicationGenericLabels
+       -> ['vcs-url', 'vcs', 'label', 'optional']
+       -> optional
+
+    maintainer_label_required
+       -> Label 'maintainer' has to be specified.
+       -> The name and email of the maintainer (usually the submitter).
+       -> https://fedoraproject.org/wiki/Container:Guidelines#LABELS
+       -> ['maintainer', 'label', 'required']
+       -> optional
+
+    io.openshift.expose-services_label
+       -> Label 'io.openshift.expose-services' has to be specified.
+       -> port:service pairs separated with comma, e.g. "8080:http,8443:https"
+       -> ?????
+       -> ['io.openshift.expose-services', 'label', 'optional']
+       -> optional
+
+    maintainer_label_required
+       -> Label 'maintainer' has to be specified.
+       -> The name and email of the maintainer (usually the submitter).
+       -> https://fedoraproject.org/wiki/Container:Guidelines#LABELS
+       -> ['maintainer', 'label', 'required']
+       -> optional
+
+    DOCKERFILE:
```

Comment:
Since we are not checking dockerfiles, this is a little bit confusing

Reply:
We can probably remove this part for now.

Reply:
The list was generated by colin ;)

Reply:
Yes, we should implement it or delete the checks to not see that...;-)